### ISSUE TITLE
Graphite template - Match tags multiple times

### ISSUE THREAD

**Issue by m4ce** [influxdata/influxdb#5416]
Would it be possible to allow tags to be matched multiple times? This is only possible for 'measurement' right now.

Reason for this is that sometimes you don't want to replace dots with any other chars (e.g. underscore) at the source. So you may have something like this:

```
metric: host1.domain.com.cpu.cpu0.iowait  template: host.host.host.measurement.resource.field
```

Thanks,
Matteo

**Comment by adrianlzt** [comment]
+1

**Comment by gunnaraasen** [comment]
@m4ce Thanks for adding this feature via https://github.com/influxdata/influxdb/pull/5419! Matching multiple tags is now available in v0.11.
